jdi-light/src/main/java/com/epam/jdi/light/elements/common/Mouse.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.jdi.light.elements.common;
+
+import com.epam.jdi.light.elements.interfaces.base.ICoreElement;
+import org.openqa.selenium.Rectangle;
+import org.openqa.selenium.WebElement;
+
+import java.awt.*;
+import java.awt.event.InputEvent;
+
+import static com.epam.jdi.light.elements.common.Keyboard.getRobot;
+import static com.epam.jdi.light.elements.composite.WebPage.zoomLevel;
+import static com.epam.jdi.tools.Timer.sleep;
+import static java.awt.event.InputEvent.BUTTON1_DOWN_MASK;
+import static java.awt.event.InputEvent.BUTTON2_DOWN_MASK;
+
+public class Mouse {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Mouse' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4958360)